extmod/nimble/modbluetooth_nimble.c

```diff
@@ -289,8 +289,15 @@ STATIC int gap_event_cb(struct ble_gap_event *event, void *arg) {
                 mp_bluetooth_gatts_on_indicate_complete(event->notify_tx.conn_handle, event->notify_tx.attr_handle, event->notify_tx.status == BLE_HS_EDONE ? 0 : event->notify_tx.status);
             }
         }
```

Comment:
Hi @jimmo the case statement above is missing the `break` 

FYI it's always missed it, "didn't matter" being the final case statement previously.

Reply:
Thanks! Fixed.